src_c/bitmask.c

```diff
@@ -88,9 +88,18 @@ static INLINE unsigned int bitcount(BITMASK_W n)
 bitmask_t *bitmask_create(int w, int h)
 {
   bitmask_t *temp;
-  temp = malloc(offsetof(bitmask_t,bits) + h*((w - 1)/BITMASK_W_LEN + 1)*sizeof(BITMASK_W));
-  if (!temp)
+  size_t size;
+
+  size = offsetof(bitmask_t,bits);
+  if (w && h) {
```

Comment:
For the case where w<=0, or h<=0 this check would pass and the size would be a weird amount.  But I guess we check that in the mask.c.